doc/arkode/guide/source/Mathematics.rst

```diff
@@ -572,6 +574,110 @@ using a fixed time-step size.
 .. The `ark_kepler.c` example demonstrates an implementation of such controller.
 
 
+.. _ARKODE.Mathematics.SplittingStep:
+
+SplittingStep -- Operator splitting methods
+================================================
+
```

Comment:
Similar question as elsewhere -- why were the new steppers inserted before MRIStep?  As everywhere else, I see no reason to jumble the order.  These should be listed chronologically: ARKStep, ERKStep, MRIStep, SPRKStep, SplittingStep/ForcingStep/LSRKStep, ...

Reply:
It made sense to group MRI, splitting, and forcing in the docs and I put the new methods first since they are simpler. I can switch to chronological, but I'll note many place put SPRKStep before MRIStep. Should I reorder SPRKStep while I'm at it?

Reply:
We can discuss this as a group, but I think that ordering them either chronologically or most-to-least-used would be optimal.  In the latter case, we have multiple applications who use ARKStep, and a few who use ERKStep and MRIStep (with more coming).  I see no reason to insert new and currently-unused steppers in front, forcing users to scroll down to get to the documentation that they need.